scripts/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "$HEROKU_API_KEY" | docker login -u "$HEROKU_LOGIN" --password-stdin registry.heroku.com
+docker build --target production --tag olutiab/olutiab-todo-app .
+docker tag olutiab/olutiab-todo-app registry.heroku.com/olutiab-todo-app/web
```

Comment:
You can add multiple tags during the build stage if that's preferable, but happy if you think this is more readable
e.g. could replace these two with:
```
docker build --target production --tag olutiab/olutiab-todo-app --tag registry.heroku.com/olutiab-todo-app/web .
```